src/accelerate/tracking.py

```diff
@@ -536,6 +537,31 @@ def log(self, values: dict, step: Optional[int], **kwargs):
         for key, value in values.items():
             self.writer.track(value, name=key, step=step, **kwargs)
 
+    @on_main_process
+    def log_images(
+        self, values: dict, step: Optional[int] = None, kwargs: Dict[str, dict] = {"aim_image": {}, "track": {}}
```

Comment:
Let's not have a mutable default argument, even if right now, nothing incorrect is happening. Instead, let's set it to `None` and in the function body, add those lines:

```python
aim_image_kw = {} if kwargs is None else kwargs["aim_image"]
track_kw = {} if kwargs is None else kwargs["track"]
```

Reply:
Thanks for the suggestion! I made a slightly change by using `get` to avoid `KeyError`.